Vizzuality/4-growth
# FE - in the sandbox, show a message when an indicator fails or has no data

## FE - in the sandbox, show a message when an indicator fails or has no data

### Designs

[Figma](https://www.figma.com/design/1MQAbB99h05eR5xbHqNDVn/4Growth-%5BInternal%5D?node-id=7046-32069&m=dev)

### Testing instructions

- navigate to `/sandbox?indicator=foo` to see no-data component (not found/404 error)
- all other server errors will render [this](https://www.figma.com/design/1MQAbB99h05eR5xbHqNDVn/4Growth-%5BInternal%5D?node-id=7046-32069&m=dev) view


### Feature relevant tickets

[Jira](https://vizzuality.atlassian.net/browse/SIRH-188)
